spark-2.0-on-suse/azuredeploy.json

```diff
@@ -44,8 +44,8 @@
     "storageAccountName": "[concat(uniquestring(resourceGroup().id),'storageac')]",
     "sparkMasterMachineName": "sparkmaster",
     "sparkSlaveMachineName": "sparkslave",
-    "sparkMasterSize": "Basic_A3",
-    "sparkSlavesSize": "Basic_A2",
+    "sparkMasterSize": "Standard_D3_v2",
+	"sparkSlavesSize": "Standard_D3_v2",
```

Comment:
There are a few places in the template where we've mixed tabs and spaces for indenting...  Can you throw this in VS Code and format it (SIFT+ALT+F) and then I think we good to go...